copy_to_hscdiip.R

```diff
@@ -0,0 +1,32 @@
+dir_folder <- "/conf/sourcedev/Source_Linkage_File_Updates"
+target_folder <- "/conf/hscdiip/copy_test"
```

Comment:
```suggestion
target_folder <- "/conf/hscdiip/"
```
When this is ready for merging back in, we need to remember to remove the test folder